### PR TITLE
Extract assets from some of the overlays.

### PR DESCRIPTION
**Pull request by louist103**

### CODE REVIEW

File: assets/xml/overlays/ovl_Arrow_Ice.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<Root>
+    <File Name="ovl_Arrow_Ice" BaseAddress="0x80867cd0" RangeStart="0x9F0" RangeEnd="0x1E10" Segment="128">    
+        <Texture Name="sTexture1" OutName="ice_tex_1" Format="ia8" Width="32" Height="64" Offset="0x9F0"/>
+        <Texture Name="sTexture2" OutName="ice_tex_2" Format="ia8" Width="32" Height="64" Offset="0x11F0"/>
```
Comment (AngheloAlf) [inline assets/xml/overlays/ovl_Arrow_Ice.xml:4]:
```suggestion
        <Texture Name="sIceArrow1Tex" OutName="ice_tex_1" Format="ia8" Width="32" Height="64" Offset="0x9F0"/>
        <Texture Name="sIceArrow2Tex" OutName="ice_tex_2" Format="ia8" Width="32" Height="64" Offset="0x11F0"/>
```

File: assets/xml/overlays/ovl_Arrow_Fire.xml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<Root>
+    <File Name="ovl_Arrow_Fire" BaseAddress="0x80865d10" RangeStart="0x9D0" RangeEnd="0x1DF0" Segment="128">    
+        <Texture Name="sFireArrowTexture1" OutName="fire_tex_1" Format="ia8" Width="32" Height="64" Offset="0x9D0"/>
+        <Texture Name="sFireArrowTexture2" OutName="fire_tex_2" Format="ia8" Width="32" Height="64" Offset="0x11D0"/>
```
Comment (AngheloAlf) [inline assets/xml/overlays/ovl_Arrow_Fire.xml:4]:
```suggestion
        <Texture Name="sFireArrow1Tex" OutName="fire_tex_1" Format="ia8" Width="32" Height="64" Offset="0x9D0"/>
        <Texture Name="sFireArrow2Tex" OutName="fire_tex_2" Format="ia8" Width="32" Height="64" Offset="0x11D0"/>
```

File: assets/xml/overlays/ovl_Arrow_Light.xml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<Root>
+    <File Name="ovl_Arrow_Light" BaseAddress="0x80869cb0" RangeStart="0xA00" RangeEnd="0x1E20" Segment="128">    
+        <Texture Name="sLightArrowTexture1" OutName="light_tex_1" Format="ia8" Width="32" Height="64" Offset="0xA00"/>
+        <Texture Name="sLightArrowTexture2" OutName="light_tex_2" Format="ia8" Width="32" Height="64" Offset="0x1200"/>
```
Comment (AngheloAlf) [inline assets/xml/overlays/ovl_Arrow_Light.xml:4]:
```suggestion
        <Texture Name="sLightArrow1Tex" OutName="light_tex_1" Format="ia8" Width="32" Height="64" Offset="0xA00"/>
        <Texture Name="sLightArrow2Tex" OutName="light_tex_2" Format="ia8" Width="32" Height="64" Offset="0x1200"/>
```

File: assets/xml/overlays/ovl_Arrow_Fire.xml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+<Root>
+    <File Name="ovl_Arrow_Fire" BaseAddress="0x80865d10" RangeStart="0x9D0" RangeEnd="0x1DF0" Segment="128">    
+        <Texture Name="sFireArrowTex1" OutName="fire_tex_1" Format="ia8" Width="32" Height="64" Offset="0x9D0"/>
+        <Texture Name="sFireArrowTex2" OutName="fire_tex_2" Format="ia8" Width="32" Height="64" Offset="0x11D0"/>
```
Comment (Dragorn421) [inline assets/xml/overlays/ovl_Arrow_Fire.xml:4]:
```suggestion
        <Texture Name="sFireArrow1Tex" OutName="fire_tex_1" Format="ia8" Width="32" Height="64" Offset="0x9D0"/>
        <Texture Name="sFireArrow2Tex" OutName="fire_tex_2" Format="ia8" Width="32" Height="64" Offset="0x11D0"/>
```

File: assets/xml/overlays/ovl_Arrow_Fire.xml [hunk 5]
```diff
@@ -0,0 +1,11 @@
+<Root>
+    <File Name="ovl_Arrow_Fire" BaseAddress="0x80865d10" RangeStart="0x9D0" RangeEnd="0x1DF0" Segment="128">    
+        <Texture Name="sFireArrowTex1" OutName="fire_tex_1" Format="ia8" Width="32" Height="64" Offset="0x9D0"/>
```
Comment (Dragorn421) [inline assets/xml/overlays/ovl_Arrow_Fire.xml:3]: Should all of these be prefixed with `s` or `g`?

File: src/overlays/actors/ovl_Bg_Ganon_Otyuka/z_bg_ganon_otyuka.c [hunk 6]
```diff
@@ -104,7 +74,7 @@ void BgGanonOtyuka_Init(Actor* thisx, GlobalContext* globalCtx) {
 
     Actor_ProcessInitChain(thisx, sInitChain);
     DynaPolyActor_Init(&this->dyna, DPM_UNK);
-    CollisionHeader_GetVirtual(&sColHeader, &colHeader);
+    CollisionHeader_GetVirtual(&sOtyukaCollision, &colHeader);
```
Comment (Dragorn421) [inline src/overlays/actors/ovl_Bg_Ganon_Otyuka/z_bg_ganon_otyuka.c:77]:
```suggestion
    CollisionHeader_GetVirtual(&sOtyukaCol, &colHeader);
```

File: src/overlays/actors/ovl_End_Title/z_end_title.c [hunk 7]
```diff
@@ -115,7 +115,7 @@ void EndTitle_Draw(Actor* thisx, GlobalContext* globalCtx) {
  * This function is never executed in normal gameplay because actor params are never set to 1
  * Produces the same results as the main draw function, except without the title cards on the end screen
  */
-void func_80B65DA8(Actor* thisx, GlobalContext* globalCtx) {
+void EndTitle_DrawNintendoLogo(Actor* thisx, GlobalContext* globalCtx) {
```
Comment (Dragorn421) [inline src/overlays/actors/ovl_End_Title/z_end_title.c:118]: Idk if this should be named, the comment says this is unused so there may be a similar function that may make better use of the name

File: assets/xml/overlays/ovl_Arrow_Fire.xml (hunk 5 above)
Comment (louist103) [reply assets/xml/overlays/ovl_Arrow_Fire.xml:3]: While they are currently accessible globally they *should* probably be static later, however ZAPD doesn't have a manual static override for anything manually extracted.

File: src/overlays/actors/ovl_End_Title/z_end_title.c (hunk 7 above)
Comment (Dragorn421) [reply src/overlays/actors/ovl_End_Title/z_end_title.c:118]: nvm I thought EndTitle was much bigger than that, but this is the only function apart from the 4 main procs

also the comment is wrong, temple of time has an actor list with `ACTOR_END_TITLE` and params 1. Maybe the scene setup using it is unused but I would be surprised

**Comment by louist103** [comment]
#985

**Pull request closed without merging**